### PR TITLE
format done on pack

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24409

## Description
Can now run format on pack and get all its files.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -100,11 +100,14 @@ def get_files_in_dir(project_dir: str, file_endings: list, recursive: bool = Tru
     :return: The path of files with file_endings in the current dir
     """
     files = []
-    pattern: str = '/**/*.' if recursive else '/*.'
+    project_path = Path(project_dir)
```
Comment (yaakovi) [inline demisto_sdk/commands/common/tools.py:103]:
```suggestion
    project_path = Path(project_dir)
    glob_function = project_path.rglob if recursive else project_path.glob
```

File: demisto_sdk/commands/common/tools.py [hunk 2]
```diff
@@ -100,11 +100,14 @@ def get_files_in_dir(project_dir: str, file_endings: list, recursive: bool = Tru
...
     for file_type in file_endings:
         if project_dir.endswith(file_type):
             return [project_dir]
-        files.extend([f for f in glob.glob(project_dir + pattern + file_type, recursive=recursive)])
+        if recursive:
+            files.extend([str(f) for f in project_path.rglob(f'*.{file_type}')])
+        else:
+            files.extend([str(f) for f in project_path.glob(f'*.{file_type}')])
```
Comment (yaakovi) [inline demisto_sdk/commands/common/tools.py:110]:
```suggestion
        files.extend([str(f) for f in glob_function(f'*.{file_type}')])
```

File: demisto_sdk/commands/format/format_module.py [hunk 3]
```diff
@@ -108,6 +108,8 @@ def format_manager(input: str = None,
                 current_excluded_files.remove('CommonServerPython.py')
             if os.path.basename(file_path) in current_excluded_files:
                 continue
+            if dirname.endswith('test_data'):
```
Comment (yaakovi) [inline demisto_sdk/commands/format/format_module.py:111]: need to skip the `doc_imgs` folder

**Review by yaakovi — APPROVED** [review]

File: demisto_sdk/commands/format/format_module.py (hunk 3 above)
Comment (orlichter1) [reply demisto_sdk/commands/format/format_module.py:111]: @yaakovi any other folders we can skip?

Comment (yaakovi) [reply demisto_sdk/commands/format/format_module.py]: nothing I can think of

**Pull request merged**